demo/js/demo.js

```diff
@@ -1,25 +1,55 @@
 (function(){
 	'use strict';
-	var colors = ['none', 'primary', 'secondary', 'success', 'warning', 'error', 'grey', 'light', 'yellow', 'green', 'orange', 'red'];
-	
+
 	// we use underscore cuz it's awesome
 	// http://underscorejs.org/
 	angular.module('underscore', []).factory('_', function () { return window._; });
+	angular.module('moment', []).factory('moment', function () { return window.moment; });
 
-	angular.module('demoApp',['ui.bootstrap']);
+	angular.module('demoApp',['lui', 'ui.bootstrap', 'ngRoute', 'ngSanitize', 'ui.select', 'ngMockE2E']);
 
 	angular.module('demoApp')
-	.controller('buttonsCtrl', ['$scope', function($scope){
-		$scope.colors = colors;
-		$scope.styles = ['default', 'flat', 'wired', 'filling'];
-		$scope.sizes = ['small', 'default', 'large', 'x-large'];
-
-		$scope.color = '';
-		$scope.style = '';
-		$scope.size = '';
-		$scope.status = '';
-		$scope.inverted = false;
-		$scope.disabled = false;
+	.controller('bannerCtrl', ['$scope', '$location', function($scope, $location) {
+		$scope.isActive = function(viewLocation) {
+			return viewLocation === $location.path();
+		};
```

Comment:
when you copy/paste code from stackoverflow, can you comment your code and put the link 
http://stackoverflow.com/questions/12295983/set-active-tab-style-with-angularjs